yt_dlp/extractor/bilibili.py

```diff
@@ -1028,12 +1068,29 @@ def _extract_video_metadata(self, url, video_id, season_id):
     def _real_extract(self, url):
         season_id, ep_id, aid = self._match_valid_url(url).group('season_id', 'ep_id', 'aid')
         video_id = ep_id or aid
+        chapters = None
+
+        if ep_id:
+            intro_ending_json = self._call_api(f'/web/v2/ogv/play/episode?episode_id={ep_id}&platform=web', video_id)
```

Comment:
This should be `fatal=False`, but call_api doesnt have the param, so you'll have to add it
```py
    def _call_api(self, path, room_id, **kwargs**):
        api_result = self._download_json(..., **kwargs)
```
```suggestion
            intro_ending_json = self._call_api(
                f'/web/v2/ogv/play/episode?episode_id={ep_id}&platform=web',
                video_id, fatal=False) or {}
```